versions-todo.md

```diff
@@ -0,0 +1,6 @@
+- autogenerate elements on draft version
```

Comment:
This commit can be rebased out, correct?

Reply:
I think the whole file can be removed and maybe added to the PR description as TODOs